ports/stm32/boards/stm32wl55_af.csv

```diff
@@ -1,45 +1,45 @@
-PortC,PC6 ,             ,                ,          ,                  ,              ,I2S2_MCK         ,       ,             ,              ,   ,    ,    ,                ,                       ,                       ,EVENTOUT,
-PortC,PC13,             ,                ,          ,                  ,              ,                 ,       ,             ,              ,   ,    ,    ,                ,                       ,                       ,EVENTOUT,
-PortC,PC14,             ,                ,          ,                  ,              ,                 ,       ,             ,              ,   ,    ,    ,                ,                       ,                       ,EVENTOUT,
-PortC,PC15,             ,                ,          ,                  ,              ,                 ,       ,             ,              ,   ,    ,    ,                ,                       ,                       ,EVENTOUT,
-PortH,PH3 ,             ,                ,          ,                  ,              ,                 ,       ,             ,              ,   ,    ,    ,                ,                       ,                       ,EVENTOUT,
+Port ,Pin ,AF0              ,AF1          ,AF2     ,AF3                  ,AF4      ,AF5              ,AF6          ,AF7                  ,AF8                   ,AF9 ,AF10,AF11,AF12        ,AF13                   ,AF14             ,AF15        ,ADC      
+     ,    ,DEBUG/RCC/RTC/SYS,LPTIM1/TIM1/2,TIM2    ,I2S2/LPTIM3/SPI2/TIM1,I2C1/2/3 ,I2S1/2/SPI1/2    ,DEBUG/RF     ,USART1/2             ,IR/LPUART1            ,    ,    ,    ,COMP1/2/TIM1,DEBUG                  ,LPTIM2/TIM2/16/17,CM4         ,ADC      
+PortA,PA0 ,                 ,TIM2_CH1     ,        ,                     ,I2C3_SMBA,I2S1_CKIN        ,             ,USART2_CTS/USART2_NSS,                      ,    ,    ,    ,COMP1_OUT   ,DEBUG_PWR/REGLP1S      ,TIM2_ETR         ,CM4_EVENTOUT,         
+PortA,PA1 ,                 ,TIM2_CH2     ,        ,LPTIM3_OUT           ,I2C1_SMBA,SPI1_SCK         ,             ,USART2_DE/USART2_RTS ,LPUART1_RTS           ,    ,    ,    ,            ,DEBUG_PWR/REGLP2S      ,                 ,CM4_EVENTOUT,         
+PortA,PA2 ,RCC_LSCO         ,TIM2_CH3     ,        ,                     ,         ,                 ,             ,USART2_TX            ,LPUART1_TX            ,    ,    ,    ,COMP2_OUT   ,DEBUG_PWR/LDORDY       ,                 ,CM4_EVENTOUT,         
+PortA,PA3 ,                 ,TIM2_CH4     ,        ,                     ,         ,I2S2_MCK         ,             ,USART2_RX            ,LPUART1_RX            ,    ,    ,    ,            ,                       ,                 ,CM4_EVENTOUT,         
+PortA,PA4 ,                 ,LPTIM1_OUT   ,        ,                     ,         ,SPI1_NSS         ,             ,USART2_CK            ,                      ,    ,    ,    ,            ,DEBUG_SUBGHZSPI/NSSOUT ,LPTIM2_OUT       ,CM4_EVENTOUT,         
```

Comment:
`DEBUG_SUBGHZSPI/NSSOUT` should be `DEBUG_SUBGHZSPI_NSSOUT`   (although this isn't currently a supported AF, the `/` shouldn't be used here).

Reply:
Ah, I've found where this change is coming from.... the script had handling to turn signals like this xml format signal `DEBUG_JTMS-SWDIO` into the form matching datasheet / existing csv `DEBUG_JTMS/SWDIO`.
That was also catching `DEBUG_SUBGHZSPI/NSSOUT` which in the xml is listed as `DEBUG_SUBGHZSPI-NSSOUT`